common/app/model/content.scala

```diff
@@ -233,7 +237,52 @@ object Content {
     }
   }
 
-  def make(apiContent: contentapi.Content): Content = {
+  object BodyBlocks {
+    def make(blocks: Option[Blocks]): Seq[BodyBlocks] =
+    blocks.toSeq.flatMap { blocks =>
```

Comment:
From my point of view, it's just too easy to skim over and assume that every reference to `blocks` is a reference to the top-level parameter, whether in Github or IntelliJ.